tartiflette/resolver_decorator.py

```diff
@@ -24,6 +38,13 @@ def __init__(self, name: str, schema=None):
         self.field = self._schema.get_field_by_name(name=name)
 
     def __call__(self, resolver, *args, **kwargs):
+        if not iscoroutinefunction(resolver):
```

Comment:
I thing it's moot to test IF we decide that in order to work in our engine, a resolver MUST/SHOULD be async ? @tsunammis what do you thing ?

Reply:
This is to filter the inputs of the user. We won't check afterwards :)